frontend/components/endpoint-page.js

```diff
@@ -0,0 +1,217 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { staticBadgeUrl } from '../lib/badge-url'
+import { baseUrl } from '../constants'
+import Meta from './meta'
+import Header from './header'
+import Footer from './footer'
+import { H3, Badge } from './common'
+import { Snippet } from './snippet'
+
+const Explanation = styled.div`
+  max-width: 800px;
+  display: block;
+`
+
+const JsonExampleBlock = styled.code`
+  display: inline-block;
+
+  text-align: left;
+  line-height: 1.2em;
+  padding: 16px 18px;
+
+  border-radius: 4px;
+  background: #eef;
+
+  font-family: Lekton;
+  font-size: ${({ fontSize }) => fontSize};
+
+  white-space: pre;
+`
+
+const JsonExample = ({ data }) => (
+  <JsonExampleBlock>{JSON.stringify(data, undefined, 2)}</JsonExampleBlock>
+)
+JsonExample.propTypes = {
+  data: PropTypes.string.isRequired,
+}
+
+const Schema = styled.dl`
+  display: inline-block;
+  max-width: 800px;
+
+  margin: 0;
+  padding: 10px;
+  text-align: left;
+
+  background: #efefef;
+
+  clear: both;
+  overflow: hidden;
+
+  dt,
+  dd {
+    padding: 0 1%;
+    margin-top: 8px;
+    margin-bottom: 8px;
+    float: left;
+  }
+
+  dt {
+    width: 100px;
+    clear: both;
+  }
+
+  dd {
+    margin-left: 20px;
+    width: 75%;
+  }
+
+  @media (max-width: 600px) {
+    .data_table {
+      text-align: center;
+    }
+  }
+`
+
+const EndpointPage = () => (
+  <div>
+    <Meta />
+    <Header />
+    <H3 id="static-badge">Endpoint (Beta)</H3>
+    <Snippet snippet={`${baseUrl}/badge/endpoint.svg?url=...&style=...`} />
+    <p>Endpoint response:</p>
+    <JsonExample
+      data={{
+        schemaVersion: 1,
+        label: 'hello',
+        message: 'sweet world',
+        color: 'orange',
```

Comment:
😆 For some reason my brain read this as `hello sweet orange`. wonder if i am hungry...